crates/swift-integration-tests/src/string.rs

```diff
@@ -8,13 +8,19 @@ mod ffi {
 
     extern "Swift" {
         fn create_swift_string() -> String;
+        fn take_and_return_swift_string(arg: String) -> String;
     }
 }
 
 fn run_string_tests() {
     let string = ffi::create_swift_string();
```

Comment:
Maybe, if this PR get merged, we could remove this from `crates/swift-integration-tests/src/string.rs`.

Reply:
Makes sense